src/megatron/bridge/recipes/minimax_m2/__init__.py

```
@@ -0,0 +1,24 @@
+# limitations under the License.
+
+from megatron.bridge.recipes.minimax_m2.minimax_m2_5 import (
+    minimax_m2_5_peft_config,
+    minimax_m2_5_sft_config,
+)
+
+
+__all__ = [
+    "minimax_m2_5_sft_config",
+    "minimax_m2_5_peft_config",
+]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)